src/components/product-page/BreadCrumbs.vue

```diff
@@ -0,0 +1,28 @@
+<script setup lang="ts">
+  import useCategory from "@/composables/api/useCategory"
+  import { capitalize } from "@/utils"
+
+  defineProps<{ productName: string }>()
+
+  const { category } = useCategory()
+</script>
+
+<template>
+  <nav>
+    <router-link class="link-primary link-animation" :to="{ name: 'home' }">
+      Home
+    </router-link>
+    <router-link class="link-primary link-animation" :to="{ name: 'category' }">
+      {{ capitalize(category?.name) }}
```

Comment:
I mean yes, but why not just use `category.name`?

Reply:
Because the category is named Keyboard kits and I don't like it